Extending/FileSystemProviders/Azure-Blob-Storage/index.md

```diff
@@ -82,32 +82,7 @@ Invoke the `.AddAzureBlobMediaFileSystem()` extention method in the `ConfigureSe
         }
 ```
 
-Next invoke `UseAzureBlobMediaFileSystem();` in the `.WithMiddleware` call, like so:
```

Comment:
This article targets both v9 and v10, so instead of removing all code, we should add a note saying this is not required in v10 anymore.